<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="menuTabMenu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1201</width>
      <height>651</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="proxyTab">
     <attribute name="title">
      <string>Proxy</string>
     </attribute>
     <widget class="QTabWidget" name="httpTabMenu">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>250</y>
        <width>1201</width>
        <height>450</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="requestTab">
       <attribute name="title">
        <string>Request</string>
       </attribute>
       <widget class="QTextEdit" name="requestText">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1201</width>
          <height>450</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="responseTab">
       <attribute name="title">
        <string>Response</string>
       </attribute>
       <widget class="QTextEdit" name="responseText">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1201</width>
          <height>450</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>1201</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>0</y>
        <width>1201</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="forwardBtn">
         <property name="text">
          <string>Forward</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dropBtn">
         <property name="text">
          <string>Drop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="interceptBtn">
         <property name="text">
          <string>Intercept</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="actionBtn">
         <property name="text">
          <string>Action</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="historyTab">
     <attribute name="title">
      <string>History</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
